docs/user-guide/configuring.md

```diff
@@ -227,6 +227,8 @@ And in YAML:
 
 These examples allow `var1` to be overwritten in your code, but disallow it for `var2`.
 
+**Note:** [no-global-assign](../rules/no-global-assign.md) rule warns modifying of variables which are disallowed overwritten.
```

Comment:
"Enable the `no-global-assign` rule to prevent overwriting of read-only global variables in your code."